### DISCUSSION TITLE
How can I export a list of the PR request

### DISCUSSION THREAD

**Discussion by mikasaa09** [community/community#66152] [Actions]
### Select Topic Area

Product Feedback

### Body

I want to export a list of the PR request from a repo, with the statuses.

Someone know how can I do that? I want to be able to build metrics for my team.

Thanks!

**Comment by seyLu** [comment]
#### Hi @Giannilopez 

You can use the Github API to extract PR info in a repo programatically.

- [Use Github REST API to interact with Pull Requests](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28)

**Reply by mikasaa09** [reply]
Is this the only way?

**Reply by seyLu** [reply]
It's not the only way.

You can also use github cli to achieve the same thing.

You can read more on:
- [Filtering and Searching issues and Pull Requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests)
- [Github CLI Manual: `gh pr list`](https://cli.github.com/manual/gh_pr_list)
- [Github CLI Manual: `gh pr status`](https://cli.github.com/manual/gh_pr_status)
